e2e-test/src/tests/analyze.test.js

```diff
@@ -0,0 +1,106 @@
+    /* eslint prefer-template: "off" */
+    await expect(page).toClickXPath("//*[text()='C26 - C26.Fol']" + vBtn({ title: 'Adds the highlighted Metabolites to the selected set' }));
+    selectedMetabolites = await page.$x("//div[@class='v-input v-input--selection-controls v-input--checkbox v-input--hide-details v-input--is-label-active v-input--is-dirty theme--light']");
+    await expect(selectedMetabolites).toHaveLength(1);
+
+    await expect(page).toClickXPath("//*[text()='C26 - Fol']" + vBtn({ title: 'Adds the highlighted Metabolites to the selected set' }));
+    selectedMetabolites = await page.$x("//div[@class='v-input v-input--selection-controls v-input--checkbox v-input--hide-details v-input--is-label-active v-input--is-dirty theme--light']");
+    await expect(selectedMetabolites).toHaveLength(4);
+
+    await expect(page).toClickXPath("//*[text()='C26 - Veh']" + vBtn({ title: 'Adds the highlighted Metabolites to the selected set' }));
+    selectedMetabolites = await page.$x("//div[@class='v-input v-input--selection-controls v-input--checkbox v-input--hide-details v-input--is-label-active v-input--is-dirty theme--light']");
+    await expect(selectedMetabolites).toHaveLength(7);
+
+    await expect(page).toClickXPath("//*[text()='C26.Fol - Fol']" + vBtn({ title: 'Adds the highlighted Metabolites to the selected set' }));
+    selectedMetabolites = await page.$x("//div[@class='v-input v-input--selection-controls v-input--checkbox v-input--hide-details v-input--is-label-active v-input--is-dirty theme--light']");
+    await expect(selectedMetabolites).toHaveLength(8);
+
+    await expect(page).toClickXPath("//*[text()='C26.Fol - Veh']" + vBtn({ title: 'Adds the highlighted Metabolites to the selected set' }));
+    selectedMetabolites = await page.$x("//div[@class='v-input v-input--selection-controls v-input--checkbox v-input--hide-details v-input--is-label-active v-input--is-dirty theme--light']");
+    await expect(selectedMetabolites).toHaveLength(11);
+
+    await expect(page).toClickXPath("//*[text()='Fol - Veh']" + vBtn({ title: 'Adds the highlighted Metabolites to the selected set' }));
+    selectedMetabolites = await page.$x("//div[@class='v-input v-input--selection-controls v-input--checkbox v-input--hide-details v-input--is-label-active v-input--is-dirty theme--light']");
+    await expect(selectedMetabolites).toHaveLength(12);
+  });
+  it('select and unselect filters', async () => {
+    await expect(page).toClickXPath(vBtn('Your Datasets'));
+
+    await uploadDataset('tableTest');
+    await expect(page).toClickXPath(vListTile({ title: 'tableTest.csv ', content: 'Dataset ready for analysis.' }) + vBtn('View Data'));
+
+    // click download data
+    await expect(page).toClickXPath(vListTile({ title: 'Download Data' }));
+
+    // unselect 'Selected' for Metabolite Filter
+    await expect(page).toClickXPath("//input[@aria-checked='true'][@aria-label='Selected (0)']");
```

Comment:
I'm not sure if this xpath is explicit enough

Reply:
Generally it is, but I don't like keying off of aria properties.
`aria` tags are special HTML tags that are used by screen readers to provide additional context about elements to people who can't see them. If you're blind and have a browser with a screen reader, the screen reader should use that extra information to read to you what your hovering over on the screen.
Try `//div[label[contains(.,'Selected (5')]]//input`